packages/grafana-llm-app/pkg/plugin/llm_provider.go

```diff
@@ -0,0 +1,129 @@
+package plugin
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+)
+
+var errBadRequest = errors.New("bad request")
+
+type Model string
+
+const (
+	ModelDefault      = "default"
+	ModelHighAccuracy = "high-accuracy"
```

Comment:
Rather than having a model named 'default' would it be better to have slightly more descriptive names and allow the user to change the default as they see fit? We could still set the default to the 'mid range' model to begin with.

Maybe this is just unnecessary complexity though - would users actually want to do that?

(I realise now this should have been in the design doc, sorry!)

Reply:
👍 yep, to keep things more in line with existing semantics, we're swapping this to small/medium/large